openapi.yaml

```diff
@@ -3504,13 +3504,25 @@ components:
       - title
       - type
       type: object
+    GoalRepositories:
+# {% if False %}
+      allOf:
+# {% else %}{{ "\n" }}      anyOf:{% endif %}
+      - $ref: '#/components/schemas/RepositorySet'
+      - items:
+          type: string
+        nullable: true
+        type: array
```

Comment:
No ambiguous types, please. Always require a list, no scalar values.

Reply:
Sorry I don't understand, valid values are
- `null`
- `[]`
- `["github.com/a"]

Reply:
No `null` should be allowed. Either a list or not sent at all. This is consistent with how the rest of the models already work.

Reply:
So `[]` means "the goals refers to every repositories" ?